src/modules/tree/style.css

```diff
@@ -0,0 +1,42 @@
+.tree,
```

Comment:
What's this file? Did you write it yourself? We want to avoid css files as much as possible.